### PR TITLE
Remove sendRequests method

### PR DESCRIPTION
**Pull request by joelwurtz**
Remove sendRequests method with new version of httplug contract

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,8 +16,7 @@
     ],
     "require": {
         "php": ">=5.5.0",
-        "php-http/httplug": "^1.0@dev",
-        "php-http/utils": "^0.1@dev",
+        "php-http/httplug": "^1.0",
```
Comment (sagikazarmark) [inline composer.json]: This won't work, since it is not stable yet.

Comment (sagikazarmark) [inline composer.json]: Actually, I am not sure.

**Comment by sagikazarmark** [comment]
Integration tests PR merged, tests should run fine.

File: composer.json (hunk 1 above)
Comment (joelwurtz) [inline composer.json]: It works because we have the minimum stability and prefere stable option in composer.json

**Comment by joelwurtz** [comment]
Is there a way to relaunch them ?

**Comment by sagikazarmark** [comment]
Build restarted, but still failing. But shouldn't hhvm be added to allowed failures?

**Comment by sagikazarmark** [comment]
Ah, it is. Then no problem.

**Pull request merged**